src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,91 @@
 package calculator;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+import static camp.nextstep.edu.missionutils.Console.readLine;
+
 public class Application {
+    public static final String CUSTOM_SEPARATOR_START = "//";
+    public static final String CUSTOM_SEPARATOR_END = "\\\\n";
+    public static final String CUSTOM_FORMAT = CUSTOM_SEPARATOR_START + "(.*?)" + CUSTOM_SEPARATOR_END + "(.*)";
+
+    public static List<String> separators = new ArrayList<>(Arrays.asList(",", ":"));
+
+    public static boolean inputIsEmpty(String input) {
```

Comment:
Java 네이밍 관습에 따라 동사로 메서드 이름 수정하면 좋을 것 같아요
```suggestion
    public static boolean isEmptyInput(String input) {
```

제 경우에는 이런 식으로 수정할 것 같네요!